contxt/__init__.py

```diff
@@ -1,19 +1 @@
-from .__version__ import (
-    __author__,
-    __author_email__,
-    __description__,
-    __license__,
-    __title__,
-    __url__,
-    __version__,
-)
-
-__all__ = [
-    "__author__",
-    "__author_email__",
-    "__description__",
-    "__license__",
-    "__title__",
-    "__url__",
-    "__version__",
-]
+__version__ = "1.0.0b7"
```

Comment:
currently, the cli consumes this version number. because an end user will not have access to the `pyproject.toml`, it has to be separate. we could

1. Don't give the cli access to a version number
2. Give the cli a version number, but try to infer it (potential solution: https://github.com/sdispater/poetry/issues/1036#issuecomment-489880822)
3. Hard-code it and use a tool to keep both up to date (current solution: https://github.com/sdispater/poetry/issues/144#issuecomment-440061951)